prow/git/git.go

```diff
@@ -258,8 +258,23 @@ func (r *Repo) CheckoutNewBranch(branch string) error {
 // Merge attempts to merge commitlike into the current branch. It returns true
 // if the merge completes. It returns an error if the abort fails.
 func (r *Repo) Merge(commitlike string) (bool, error) {
+	return r.MergeWithStrategy(commitlike, prowgithub.MergeMerge)
+}
+
+// MergeWithStrategy attempts to merge commitlike into the current branch given the merge strategy.
+// It returns true if the merge completes. It returns an error if the abort fails.
+func (r *Repo) MergeWithStrategy(commitlike string, mergeStrategy prowgithub.PullRequestMergeType) (bool, error) {
 	r.logger.Infof("Merging %s.", commitlike)
-	co := r.gitCommand("merge", "--no-ff", "--no-stat", "-m merge", commitlike)
+	mergeFlag := ""
```

Comment:
var mergeFlag string

Reply:
Since I was the one originally adding this and we do not have any linting for it Is there a styleguide or sth so contributors can do this right without needing feedback of a reviewer first?

Reply:
https://dave.cheney.net/practical-go/presentations/qcon-china.html#_use_a_consistent_declaration_style

Reply:
Okay, that is now something I in particular know but not others. Would you mind adding that to `CONTRIBUTING.md`?

Reply:
Where do you think it would be appropriate? I'm not sure adding "read all these docs before you contribute" is a great/necessary experience?

AKA I'm fine with someone learning these things over time, rather than requiring this kind of behavior.